src/lib/components/transaction-card.svelte

```diff
@@ -23,7 +23,7 @@
 	let openTransaction = $state(false)
 
 	const numOccurences = $derived(
```

Comment:
Please also fix the typo in the variable name:
```suggestion
	const numOccurrences = $derived(
```